README.md

```diff
@@ -48,22 +46,32 @@ composer update
 
 ## Usage
 
-The use of composer requies the usage of `require_once('/vendor/autoload.php');`
+The use of composer requires the usage of `require_once('/vendor/autoload.php');`
 
-Instance the class `SDK(hashKey, server)`
+Import the SDK use `use UDT\SDK\SASDK;`;
 
-There you can do:
+Initialize the static class `SASDK::init(hashKey, server);`
 
-- encrypt
-    - Encrypt string with 3des algorithm.
-- decrypt
-    - Decrypt string with 3des algorithm.
-- is_authentic_request
-    - Check api/token to see if request is authentic.
-- handlePayload
-    - Make a request to UDT, validate data in/out.
+There you can do:
 
-Do the following to encrypt/decrypt some key:
+- encryptSDK
+  - Encrypt string with 3des algorithm.
+- decryptSDK
+  - Decrypt string with 3des algorithm.
+- decryptUDT
+  - Decrypt string with UDT algorithm.
+- validateRequestHeaders
+  - Check api/token to see if request is authentic.
+- formatMoney
+  - Gives the standard format to money.
+- createPayment
+  - Create an order and retrieve payment url.
+- cancelOrder
+  - Cancel a pending order on UDT.
+- refundOrder
+  - Refund a paid order on UDT.
+
+Do the following on project root folder to use the key utilities:
```

Comment:
Dedazo: on the project's root folder

Reply:
No se usa plural ni indicativo en estos casos.

Reply:
No es plural, es un posesivo. Y el _the_ es simplemente un artículo que transforma el sustantivo _project_ a que sea definido y común; no cambia la conjugación del verbo ni nada. Pues, la traducción literal de como está ahorita al español es "Haz lo siguiente en el directorio raíz proyecto" cuando debería ser "el directorio raíz del proyecto".